rest-api/test/unit_test/dao_test/participant_dao_test.py

```diff
@@ -149,7 +149,8 @@ def test_update_no_expected_version_with_ps(self):
     ps = self.participant_summary_dao.get(1)
     expected_ps = self._participant_summary_with_defaults(
```

Comment:
This tests the insert case. Let's add another check for the update case?

Reply:
done